data/curated/border-patrol/meta.yaml

```
@@ -0,0 +1,19 @@
+title: U.S. Customs and Border Protection (CBP) encounter data
```

```suggestion
title: U.S. Customs and Border Protection (CBP) Encounter Data
```
I like to use Title Case for these.